src/utils/hash.hpp

```diff
@@ -0,0 +1,60 @@
+//========================================================================================
+// Athena++ astrophysical MHD code
+// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+// Licensed under the 3-clause BSD License, see LICENSE file for details
```

Comment:
Is this file derived from code in `athena++`? Otherwise I'm not sure we need the separate Athena copyright.

Reply:
Removed, think it just came from copying another header.